lib/ansible/module_utils/net_tools/nios/api.py

```diff
@@ -222,29 +238,24 @@ def run(self, ib_obj_type, ib_spec):
         # get object reference
         ib_obj_ref, update, new_name = self.get_object_ref(self.module, ib_obj_type, obj_filter, ib_spec)
 
-        proposed_object = {}
```

Comment:
There's seems to be many changes that I am unable to understand, from this down below can u comment that why you have made all the changes. Thanks